### PR TITLE
James 1720 Upgrade Cassandra to 3.0.5

### PR DESCRIPTION
**Pull request by chibenwa**
Needs PR on Ci to work

I validated deployement tests on my computer.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -110,7 +110,7 @@ $ docker run -v $HOME/.m2:/root/.m2 -v $PWD:/origin \
 ## Howto ?
 You need a running **cassandra** in docker. To achieve this run :
 ```bash
-$ docker run -d --name=cassandra cassandra:2.2.3
+$ docker run -d --name=cassandra cassandra:3.0.5
```
Comment (mbaechler) [inline README.txt]: nope, I agree to change driver version to 3, no cassandra itself

Comment (chibenwa) [inline README.txt]: Ok, I validate it still works...

**Comment by mbaechler** [comment]
test this please

**Comment by chibenwa** [comment]
Closed because timezones are not well supported by ZonedDateTime codec, so no need to upgrade.

See https://datastax-oss.atlassian.net/browse/JAVA-1161

**Comment by mbaechler** [comment]
Actually, I don't see what you missed into ZonedDateTime codec

**Pull request closed without merging**